luigi/scheduler.py

```diff
@@ -224,7 +224,7 @@ class Worker(object):
     Structure for tracking worker activity and keeping their references.
     """
 
-    def __init__(self, worker_id, last_active=None):
+    def __init__(self, worker_id, last_active=time.time()):
```

Comment:
yes, erik is right. Sorry, my mistake, I was thinking __init__(self, worker, last_active=time.time()) would initialise last_active at object creation time. Shall I submit another pull request for a fix?